### DISCUSSION TITLE
Canale youtube

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#268] [Idee e proposte]
Buongiorno,sarebbe bello se ci fosse un canale youtube dedicato al progetto giuaschool con tutti i video-tutorial , faciliterebbe il lavoro a tutti  e non ci sarebbe bisogno di mandare l'email per qualsiasi dubbio o incertezza

**Comment by manuelbarberi** [comment]
Avevo pensato anche a questo.... Magari se @trinko lo permette, potrei anche occuparmene io personalmente... Diventa tutto più difficile con le mail.

**Reply by trinko** [reply]
> Avevo pensato anche a questo.... Magari se @trinko lo permette, potrei anche occuparmene io personalmente... Diventa tutto più difficile con le mail.

Certamente, anzi ti ringrazio per l'iniziativa.

**Comment by manuelbarberi** [comment]
> Buongiorno,sarebbe bello se ci fosse un canale youtube dedicato al progetto giuaschool con tutti i video-tutorial , faciliterebbe il lavoro a tutti e non ci sarebbe bisogno di mandare l'email per qualsiasi dubbio o incertezza

https://www.youtube.com/channel/UCVmP0GavypniGv7MapL84fw - questo è il link...

**Comment by manuelbarberi** [comment]
Penso da oggi
Il giorno lun 6 giu 2022 alle 14:57 gmontalbano5ct ***@***.***>
ha scritto:

**Comment by sempliciniluigi89** [comment]
@manuelbarberi Già a breve ci saranno i video-tutorial?
Ho diverse idee
1.Video tutorial scrutinio classi prime
2.Video tutorial classi seconde (con certificato delle competenze)
3.Video configurazione email in modo da ricevere le credenziali via email
4.attribuzione materie ai docenti
5.attribuzione classi ai docenti

**Comment by manuelbarberi** [comment]
Si

Il giorno lun 6 giu 2022 alle 14:59 sempliciniluigi89 <
***@***.***> ha scritto:

**Comment by sempliciniluigi89** [comment]
6.Video tutorial installazione registro sia con altervista che con x10hosting
7.Video tutorial aggiornamento registro

**Comment by manuelbarberi** [comment]
Si

Il giorno lun 6 giu 2022 alle 15:02 gmontalbano5ct ***@***.***>
ha scritto:

**Comment by manuelbarberi** [comment]
Essendo uno studente proverò ad ottimizzare al massimo i tempi… oggi uscirà
il video di installazione/aggiornamento

Il giorno lun 6 giu 2022 alle 15:25 Antonello Dessì <
***@***.***> ha scritto:

**Comment by manuelbarberi** [comment]
Non so.... l'audio si sentiva malissimo... ora li registro
Per domani a questo punto

**Comment by manuelbarberi** [comment]
Ah ok... ora edito e pubblico

**Comment by sempliciniluigi89** [comment]
Sono in ansia per questo primo video  su youtube , non vedo l'ora

**Comment by sempliciniluigi89** [comment]
@manuelbarberi come procede la preparazione del video?

**Comment by sempliciniluigi89** [comment]
Buongiorno, ma ieri non dovevano essere rilasciati i video sul nuovo canale YouTube di Giuaschool?

**Reply by manuelbarberi** [reply]
Dovrebbero uscire oggi... salvo imprevisti

**Comment by sempliciniluigi89** [comment]
verso che ora più o meno?

**Comment by sempliciniluigi89** [comment]
Scusate ma il video?

**Reply by manuelbarberi** [reply]
Ho provato fino a poco fa a caricarlo... problemi di connessione

**Reply by manuelbarberi** [reply]
Vediamo se stanotte ci lavoro e lo carico già da stanotte

**Comment by sempliciniluigi89** [comment]
Dobbiamo aspettare ancora tanto?

**Comment by sempliciniluigi89** [comment]
@manuelbarberi Riscontra ancora problemi nel caricare i video?

**Comment by sempliciniluigi89** [comment]
Buongiorno, ci sarebbe per caso qualcun altro disposto a pubblicare i video dell'installazione/aggiornamento del registro giuaschool sul canale youtube?Lo chiedo semplicemente perché i miei colleghi stanno aspettando ormai da lunedì come da voi promesso e oggi siamo a giovedì e non è uscito ancora niente, nemmeno un video.Grazie

**Comment by sempliciniluigi89** [comment]
@gmontalbano5ct questi sono i video che potrebbe fare
1.Video tutorial scrutinio classi prime
2.Video tutorial classi seconde (con certificato delle competenze)
3.Video configurazione email in modo da ricevere le credenziali via email
4.attribuzione materie ai docenti
5.attribuzione classi ai docenti
6.Importazione utenti dal file csv

**Comment by sempliciniluigi89** [comment]
@gmontalbano5ct Pensa siano fattibili questi video tutorial ?

**Comment by sempliciniluigi89** [comment]
Potrebbe procedere?

**Comment by manuelbarberi** [comment]
Ora me ne occupo io… esco da scuola e vedo

Il giorno gio 9 giu 2022 alle 13:24 gmontalbano5ct ***@***.***>
ha scritto:

**Reply by manuelbarberi** [reply]
Dammi l'ok così la tolgo da qui poi

**Comment by sempliciniluigi89** [comment]
@gmontalbano5ct Quali video ha già registrato'

**Comment by sempliciniluigi89** [comment]
@gmontalbano5ct Ci sono novità?

**Comment by sempliciniluigi89** [comment]
@manuelbarberi A che punto stanno i video?

**Comment by sempliciniluigi89** [comment]
@gmontalbano5ct Buongiorno,se non sbaglio ,lei sta aspettando ancora di ricevere l'indirizzo email di @manuelbarberi  in modo da inviargli i video ormai pronti da caricare¡

**Comment by manuelbarberi** [comment]
***@***.***


Il giorno gio 9 giu 2022 alle 19:07 gmontalbano5ct ***@***.***>
ha scritto:

**Comment by sempliciniluigi89** [comment]
Inoltre ci servirebbe urgentemente  un video tutorial sullo scrutinio della classe seconda con la gestione del certificato delle competenze

**Reply by trinko** [reply]
> Inoltre ci servirebbe urgentemente un video tutorial sullo scrutinio della classe seconda con la gestione del certificato delle competenze

Segnalo che ho pubblicato nella documentazione il manuale operativo degli scrutini:
[Pagina dei manuali](https://trinko.github.io/giuaschool-docs/user-manual.html)

**Comment by sempliciniluigi89** [comment]
Se si riuscisse  a caricare il tutto oggi sarebbe meglio,anche perché oggi stesso iniziano gli scrutini finali

**Comment by sempliciniluigi89** [comment]
@trinko Potrebbe almeno lei provare a caricare i video tutorial sul canale youtube  che è ormai da una settimana che stiamo aspettando ?Grazie

**Reply by trinko** [reply]
No, mi spiace. Non ho il tempo di occuparmi anche di questo.

**Comment by sempliciniluigi89** [comment]
Scusate , ma io non sto capendo una cosa:non era @manuelbarberi  che si era proposto di occuparsi del canale youtube del progetto Giuaschool con tutti i video al suo  interno?

**Comment by sempliciniluigi89** [comment]
É urgente la richiesta dei video tutorial

**Comment by sempliciniluigi89** [comment]
Buongiorno ,ma nessuno vuole caricare  i video tutorial del registro GIUASCHOOL sul nuovo canale youtube? A questo punto tanto vale cancellarlo ,scusatemi ,ma è passata ormai una settimana e nessuno risponde ne tanto meno carica sul canale i video richiesti .

**Comment by sempliciniluigi89** [comment]
Gentile @gmontalbano5ct  dal momento che ha già i video tutorial pronti ,non è che potrebbe creare lei un canale youtube e pubblicarli?Ovviamente con il consenso da parte di @trinko

**Comment by sempliciniluigi89** [comment]
@manuelbarberi  novità?

**Comment by sempliciniluigi89** [comment]
Gentile @manuelbarberi , come procede con i video?

**Comment by sempliciniluigi89** [comment]
Gentile @gmontalbano5ct , mi potrebbe inviare se è possibile i videotutorial  via mail?Le lascio qui sotto il mio indirizzo mail

sempliciniluigi89@gmail.com

**Comment by sempliciniluigi89** [comment]
No,purtroppo
